components/Sidebar/ClientSidebar.tsx

```
@@ -983,13 +1311,16 @@ function ClientSidebarContent({
           __sidebarChatUpdater?: (
             title: string,
             moduleId: string | null,
-            forceOldest?: boolean
+            forceOldest?: boolean,
+            currentPath?: string
           ) => string;
+          __hasPendingOptimisticChat?: boolean;
         };
         delete win.__sidebarChatUpdater;
+        delete win.__hasPendingOptimisticChat;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)